src/promptflow/tests/executor/unittests/batch/test_result.py

```diff
@@ -162,3 +168,19 @@ def test_invalid_api_calls(self, api_call):
         assert batch_result.system_metrics.total_tokens == 0
         assert batch_result.system_metrics.completion_tokens == 0
         assert batch_result.system_metrics.prompt_tokens == 0
+
+    def test_get_aggr_error_summary(self):
```

Comment:
How about e2e tests?

Reply:
After @codingrabbitt1 completes this task (https://msdata.visualstudio.com/Vienna/_workitems/edit/2699345/?view=edit), we can add an e2e test. Now if an aggregation node fails, an exception will be thrown.

Reply:
Added.